aws/cloudformation/cloud_formation_stack.yml.erb

```diff
@@ -26,8 +26,11 @@ self.log_resource_filter.push %w(FrontendLaunchConfig ASGCount)
 ---
 AWSTemplateFormatVersion: 2010-09-09
 Description: AWS CloudFormation Template for Code.org application
-<% if environment == :adhoc -%>
 Parameters:
+  InstanceType:
+    Type: String
+    Default: <%=INSTANCE_TYPE%>
```

Comment:
Parameters allow values to be overridden (either via environment variables on stack creation, or as `CDO` attributes set via `locals.yml`, chef-config `globals.yml`, etc), and persist across stack updates.